packages/design/src/icons/getIcon.ts

```diff
@@ -0,0 +1,113 @@
+import classnames from "classnames";
```

Comment:
This whole file has a bunch of different mappings in it. 
Consider looking for a way to define these as pure JS data structures (JSON?) or config file formats (yaml? lol barf). Even moving them to be defined that way and imported would help their portability.

Reply:
I tried to change as little as possible in the original files, so most of what you see here is basically just filed moved from the `components/Icon` to `design`.

For the `classnames` mapping, that's the way it's done everywhere in Atlantis. I believe extracting them requires a conversation to make sure we're consistent with the rest of packages.

The other mapping is `mapToCorrectIcon`, which is a tiny one that we can create a hash map for it. I don't think moving a small map like that to a different file would give us much benefit. Maybe we can make it an inline hash map.

Let me know what you think.

Reply:
Ok, no worries. When/if we need to, we can bring these definitions out.